MSProject/ViewControllers/LibraryTableView/ViewModel/WBLibraryViewModel.swift

```diff
@@ -74,9 +72,15 @@ class WBLibraryViewModel {
     
     func loadRents() -> SignalProducer<[WBRent], RepositoryError> {
         return self.repository.getRents().on(failed: { _ in  }, value: { value in
-            
-            let rentedBooks = value.map { $0.book!.id }
-            WBBooksManager.sharedIntance.bookViewModels.value.forEach { $0.rented =  rentedBooks.contains($0.book.id) ? true : false }
+            for rent in value {
```

Comment:
you can use foreach here too, just to be consistent, there is nothing wrong with for loops (although i think foreach is more performant if you are not doing early returns)